providers/azure/README.md

```diff
@@ -272,6 +274,133 @@ To verify that virtual kubelet has started, run:
+  ```cli
+    export VNET_RANGE=10.8.0.0/8  
+    export CLUSTER_SUBNET_RANGE=10.240.0.0/16 
+    export ACI_SUBNET_RANGE=10.241.0.0/16 
+    export VNET_NAME=myAKSVNet 
+    export CLUSTER_SUBNET_NAME=myAKSSubnet 
+    export ACI_SUBNET_NAME=myACISubnet 
+    export AKS_CLUSTER_RG=myresourcegroup 
+  ```
+  Run the following command to create a virtual network within Azure, and a subnet within that VNet. The subnet will be dedicated to the nodes in the AKS cluster.
+
+    ```cli
+    az network vnet create \
+    --resource-group $AKS_CLUSTER_RG \
+    --name $VNET_NAME \
+    --address-prefixes $VNET_RANGE \
+    --subnet-name $CLUSTER_SUBNET_NAME \
+    --subnet-prefix $CLUSTER_SUBNET_RANGE
+    ```
+
+Create a subnet that will be delegated to just resources within ACI, note that this needs to be an empty subnet, but within the same VNet that you already created. 
+
+```cli
+az network vnet subnet create \
+    --resource-group $AKS_CLUSTER_RG \
+    --vnet-name $VNET_NAME \
+    --name $ACI_SUBNET_NAME \
+    --address-prefix $ACI_SUBNET_RANGE
+```
+
+### Create a service principal
+
+Create an Azure Active Directory service principal to allow AKS to interact with other Azure resources. You can use a pre-created service principal too. 
+
+```cli
+az ad sp create-for-rbac --skip-assignment
+```
+
+The output should look similar to the following. 
+ 
+```console
+{
+  "appId": "bef76eb3-d743-4a97-9534-03e9388811fc",
+  "displayName": "azure-cli-2018-08-29-22-29-29",
+  "name": "http://azure-cli-2018-08-29-22-29-29",
+  "password": "1d257915-8714-4ce7-a7fb-0e5a5411df7f",
+  "tenant": "72f988bf-86f1-41af-91ab-2d7cd011db48"
+}
+```
+Save the output values from the command output in enviroment variables. 
+
+```cli
+export AZURE_TENANT_ID=<Tenant>
+export AZURE_CLIENT_ID=<AppId>
+export AZURE_CLIENT_SECRET=<Password>
+```
+
+Grab the virtual network resource id with the following command:
+
+```cli
+az network vnet show --resource-group $AKS_CLUSTER_RG --name $VNET_NAME --query id -o tsv
+```
+
+Grant access to the AKS cluster to use the virtual network by creating a role and assigning it. 
+
+```cli
+az role assignment create --assignee $AZURE_CLIENT_ID --scope <vnetId> --role Contributor
+```
+
+### Create an AKS cluster with a virtual network
+
+Grab the id of the cluster subnet you created earlier with the following command. 
+
+```cli
+az network vnet subnet show --resource-group $AKS_CLUSTER_RG --vnet-name $VNET_NAME --name $CLUSTER_SUBNET_NAME --query id -o tsv
+```
+Save the output in the following enviorment variable. 
+```cli 
+export VNET_SUBNET_ID=<subnet-id>
+```
+Use the following command to create an AKS cluster with the virtual network you've already created. Use the enviorment variables from the service principal output, 
+```cli
+az aks create \
+    --resource-group myResourceGroup \
+    --name myAKSCluster \
+    --node-count 1 \
+    --network-plugin azure \
+    --service-cidr 10.0.0.0/16 \
+    --dns-service-ip 10.0.0.10 \
```

Comment:
define the 10.0.0.10 in a var. we need to reference it below in "Deploy Virtual Kubelet" section as "$KUBE_DNS_IP".